### ISSUE TITLE
Enhance Readme Documentation

### ISSUE THREAD

**Issue by ishaanbedi** [ishaanbedi/webnotes#2]
Currently, the documentation in the Readme file provides an overview of the application, not a complete guide about the application or re-producing it. 

Looking forward to improving the Readme file and adding more context like schema of the backend, environment variables, setting up locally, deploying it etc.

Some references:

- Environment Variables:
```
APP_SECRET="XXXX"
NEXT_PUBLIC_APP_URL="https://webnotes.vercel.app"
NEXT_PUBLIC_SHARE_SECRET="XXXX"
XATA_API_KEY="XXXX"
XATA_FALLBACK_BRANCH="main"
```

- Backend Schema:
<img width="358" alt="Screenshot 2023-10-09 at 11 12 09 AM" src="https://github.com/ishaanbedi/webnotes/assets/39641326/cbe04519-78ec-4d01-ad7a-30b2bb6b9cd2">

**Comment by SuryaCreatX** [comment]
Can you assign me this issue?

**Comment by ishaanbedi** [comment]
> Can you assign me this issue?

Sure! Assigned. Do let me know if you need some help or further information at any point in time.

**Comment by ishaanbedi** [comment]
Hi @SuryaCreatX !
There were some recent changes like the addition of a popup on the first visit and showing instructions to download, which would be great to include in the readme file. You can refer to [closed PRs](https://github.com/ishaanbedi/webnotes/pulls?q=is%3Apr+is%3Aclosed) for the same.

**Comment by ishaanbedi** [comment]
Hi @SuryaCreatX !
Do let me know if you are facing some issues setting up the project or any other problem you might be facing.

**Comment by thecybercomrade** [comment]
Hi @ishaanbedi, can you kindly assign this Issue to me ? I will update the documentation with in this week, thanks

**Comment by ishaanbedi** [comment]
> Hi @ishaanbedi, can you kindly assign this Issue to me ? I will update the documentation with in this week, thanks

Hi @thecybercomrade !
Sure, go ahead!
Do let me know if you need any kind of help or further information.

**Comment by thecybercomrade** [comment]
Hi @ishaanbedi thank you

**Comment by ishaanbedi** [comment]
Hey @thecybercomrade !
If you are still looking forward to contributing, there have been some significant changes to the repo.
Please refer to this: https://webnotes.vercel.app/shared/cmlllfpjpd5kdkk1rd10 for all the info. regarding what has been done and what has to be done.

**Comment by ishaanbedi** [comment]
Closed as fixed by @Aadityagoel15 via #11
